tests/Microsoft.Data.DataFrame.Tests/DataFrameTableTests.cs

```diff
@@ -0,0 +1,126 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
We may need a little more tests in this PR. This doesn't seem like enough given the functionality in the library with this PR.

Reply:
I sort of agree. With this patch, the only ops allowed are things like floatColumn + float, so the scope is pretty narrow. Would you consider waiting till the next patch(will be up in an hour) where I've added a couple more tests for binary ops? That patch tests different functions and more combinations such as int + bool/double/decimal

Reply:
This is OK for now. Let's be sure to add adequate tests in future PRs.